osidb/models.py

```diff
@@ -807,6 +807,25 @@ def _validate_no_placeholder(self):
                 "OSIDB does not support write operations on placeholder flaws"
             )
 
+    def _validate_notabug_flaw_affected(self):
+        """
+        Alerts in case flaw is closed as NOTABUG but have affected components
+        """
+        if (
+            self.state != Flaw.FlawState.CLOSED
+            or self.resolution != FlawResolution.NOTABUG
+        ):
+            return
+
+        affected = self.affects.filter(
+            affectedness=Affect.AffectAffectedness.AFFECTED
+        ).first()
+        if affected:
+            self.alert(
+                "notabug_affect_ps_component",
+                f"{affected.ps_component} is affected by a flaw solved as NOTABUG.",
```

Comment:
The component is not enough to uniquely identify the affect - from the user point of view here. It is usual that there are multiple affects with the same component but the different modules. There should be both the PS module and component in the warning message.

Reply:
makes sense, will add a fix.